packages/gitbook/src/components/DocumentView/Table/RecordColumnValue.tsx

```diff
@@ -2,7 +2,7 @@ import { ContentRef, DocumentBlockTable } from '@gitbook/api';
 import { Icon } from '@gitbook/icons';
```

Comment:
As this is a bigger change, I made sure to test a bunch of different docs with differing table setups. For us internally, here's also a quick before/after comparison of some customer docs shared with us thru support when they raised this issue: https://linear.app/gitbook-x/issue/RND-3264/table-columns-sizing-doesnt-match-published-content-vs-editor#comment-b5f07348

In the cases i could check, this PR fixed it for all.